subtensor-api/src/subtensorapi/cli.py

```diff
@@ -0,0 +1,71 @@
+# The MIT License (MIT)
+# Copyright © 2022 Opentensor Foundation
```

Comment:
also, rename this file to something different, this is not going to be a CLI, right?